index.js

```diff
@@ -252,3 +266,28 @@ module.exports.percyScreenshot = async function percyScreenshot(driver, name, op
 module.exports.isPercyEnabled = async function isPercyEnabled() {
   return await utils.isPercyEnabled();
 };
+
+module.exports.slowScrollToBottom = async (driver, timeInSeconds = SCROLL_DEFAULT_SLEEP_TIME) => {
+  let scrollHeight = Math.min(await driver.executeScript('return document.documentElement.scrollHeight'), 25000);
+  const clientHeight = await driver.executeScript('return document.documentElement.clientHeight');
+  let current = 0;
+
+  let page = 1;
+  // Break the loop if maximum scroll height 25000px is reached
+  while (scrollHeight > current && current < CS_MAX_SCREENSHOT_LIMIT) {
+    current = clientHeight * page;
+    page += 1;
+    await driver.executeScript(`window.scrollTo(0, ${current})`);
+    await new Promise(resolve => setTimeout(resolve, timeInSeconds * 1000));
+
+    // Recalculate scroll height for dynamically loaded pages
+    scrollHeight = await driver.executeScript('return document.documentElement.scrollHeight');
+  }
+  // Get back to top
+  await driver.executeScript('window.scrollTo(0, 0)');
+  let sleepAfterScroll = 1;
```

Comment:
default sleep? variable?

Reply:
this is sleep after we get back to top

Reply:
since we are reseting it based on env variable it is variable(let)